temp_examples/scenic-streetmap-overlay/scenic_plot.html

```diff
@@ -0,0 +1,398 @@
+         */
+        
+        // custom info control
+        var info = L.control();
+        info.onAdd = function (map) {
+          this._div = L.DomUtil.create('div', 'info'); // create a div with a class "info"
+          this.update();
+          return this._div;
+        };
+
+        // method to update the control based on feature properties passed
+        info.update = function (props) {
+          var rollover_html=``;
+          if ((!props)&&(!now_showing)) {
+            rollover_html=`<h4>Click on any area <br />to see region info</h4>`;
+          } else if ((props)&&(!now_showing)) {
+            rollover_html+=`<h4>Show info for `+props.lsoa11cd+` (`+props.lsoa11nm+`)</h4>`;	
+          } else {
+            if (props) { // currently hovering over
+              rollover_html+=`<h4>Show info for `+props.lsoa11cd+` (`+props.lsoa11nm+`) next?</h4>`;
+            }
+            var showingCode = now_showing.lsoa11cd;
+            var showingName = now_showing.lsoa11nm;
+            var scenicness = getScenicBar(now_showing.scenic_rating);
+            var health = getHealthBar(now_showing.deprivation_health_deprivation_and_disability_score);
+            var income = getIncomeBar(now_showing.deprivation_income_score_rate);
+            var employ = getEmploymentBar(now_showing.deprivation_employment_score_rate);
+            rollover_html+=`<br/><b>Area: ${showingCode} (${showingName})</b>
+                           <br/>Scenicness<br/><div class="band" style="width:${160-scenicness*10*1.6}px; 
+                                                                        border-left:${scenicness*10*1.6}px solid #a42e3d">&nbsp;</div>
+                           Health<br/><div class="band" style="width:${160-health*10*1.6}px; border-left:${health*10*1.6}px solid #a42e3d">&nbsp;</div>
+                           Income<br/><div class="band" style="width:${160-income*10*1.6}px; border-left:${income*10*1.6}px solid #a42e3d">&nbsp;</div>
+                           Employment<br/><div class="band" style="width:${160-employ*10*1.6}px; border-left:${employ*10*1.6}px solid #a42e3d">&nbsp;</div>`;
+          }
+          this._div.innerHTML =  rollover_html;
+        };
+        info.addTo(leafletMap);
+
+        function getCrimeColour(crime_count){
+          /**
+           * Get the appropariate colour of given number of crimes according
+           * to prespecified bins (red- high counts, blue - low counts).
+           *
+           * @param  {Number} crime_count Mean num of monthly crime counts
+           * @return {String}             Hex colour for category of counts
+           */
+          return crime_count >= 50 ? '#a42e3d' :
+            crime_count >= 20  ? '#cb7572' :
+            crime_count >= 10  ? '#7fa4b4' :
+            crime_count >= 5   ? '#0e4b8d' :
+            '#012043';
+        }
+        function getScenicBar(s_rate) {
```

Comment:
Am I right that the max is being mapped to 10 and the min to 0? As this is a little confusing. Perhaps here, as opposed to saying low is 0 and high is 10, it should instead say min and max and what those values are. Just so, no one things an area got a scenic rating of 10. As in reality that only happens in Scotland! :)